handlers/update_resource_test.go

```diff
@@ -66,14 +66,14 @@ func TestUpdateInvalidResource(t *testing.T) {
 	r.PATCH("/v1/resource/100").
 		SetHeader(gofight.H{"Content-Type": "application/json"}).
 		SetJSON(gofight.D{
-			"externalIdentifier": "oo000oo0001",
-			"@context":           "http://example.com", // This is not a valid context
-			"@type":              "http://sdr.sul.stanford.edu/models/sdr3-object.jsonld",
-			"access":             "world",
-			"label":              "My work",
-			"preserve":           true,
-			"publish":            true,
-			"sourceId":           "bib12345678"}).
+			"id":       "oo000oo0001",
```

Comment:
I'm not sure why you are changing this field.  This record is invalid because the `@context` is wrong.  If you change it so it is invalid in two ways, we can't be sure this case is tested.

Reply:
That's odd. I don't recall even touching that. Will pull the existing back in and see what's up.